Synthesis failed for java-common-protos

Hello! Autosynth couldn't regenerate java-common-protos. :broken_heart:

Here's the output from running `synth.py`:

```
:41:42,237 autosynth [DEBUG] > Running: git checkout autosynth-175
Switched to branch 'autosynth-175'
2020-09-08 14:41:42,242 autosynth [INFO] > Running synthtool
2020-09-08 14:41:42,242 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-08 14:41:42,242 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/175/sponge_log.log
2020-09-08 14:41:42,243 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-08 14:41:42,461 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-175
nothing to commit, working tree clean
2020-09-08 14:41:42,594 synthtool [DEBUG] > Ensuring dependencies.
2020-09-08 14:41:42,604 synthtool [DEBUG] > Cloning googleapis.
2020-09-08 14:41:43,124 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-08 14:41:58,584 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.32.2.zip) = 5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
Loading: 0 packages loaded
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:333:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 15.451s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-08 14:41:58,647 autosynth [ERROR] > Synthesis failed
2020-09-08 14:41:58,647 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0b20f1 deps: update dependency io.grpc:grpc-bom to v1.31.1 (#16)
2020-09-08 14:41:58,654 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-08 14:41:58,659 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9f4d86d2-e590-420f-9223-70f186dccfee/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
:28:32,419 autosynth [DEBUG] > Running: git checkout autosynth-175
Switched to branch 'autosynth-175'
2020-09-09 01:28:32,425 autosynth [INFO] > Running synthtool
2020-09-09 01:28:32,425 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-09 01:28:32,425 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/175/sponge_log.log
2020-09-09 01:28:32,427 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-09 01:28:32,646 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-175
nothing to commit, working tree clean
2020-09-09 01:28:32,779 synthtool [DEBUG] > Ensuring dependencies.
2020-09-09 01:28:32,790 synthtool [DEBUG] > Cloning googleapis.
2020-09-09 01:28:33,271 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-09 01:28:48,198 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.32.2.zip) = 5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:333:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 14.912s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-09 01:28:48,257 autosynth [ERROR] > Synthesis failed
2020-09-09 01:28:48,257 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0b20f1 deps: update dependency io.grpc:grpc-bom to v1.31.1 (#16)
2020-09-09 01:28:48,263 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-09 01:28:48,269 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d3daad42-2df9-4cb8-a985-54c177a225aa/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
utosynth-175'
2020-09-09 14:28:54,486 autosynth [INFO] > Running synthtool
2020-09-09 14:28:54,487 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-09 14:28:54,487 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/175/sponge_log.log
2020-09-09 14:28:54,489 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-09 14:28:54,728 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-175
nothing to commit, working tree clean
2020-09-09 14:28:54,874 synthtool [DEBUG] > Ensuring dependencies.
2020-09-09 14:28:54,885 synthtool [DEBUG] > Cloning googleapis.
2020-09-09 14:28:55,351 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-09 14:29:13,863 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.32.2.zip) = 5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:333:1
Loading: 0 packages loaded
    currently loading: google/api
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 18.496s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-09 14:29:13,941 autosynth [ERROR] > Synthesis failed
2020-09-09 14:29:13,941 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0b20f1 deps: update dependency io.grpc:grpc-bom to v1.31.1 (#16)
2020-09-09 14:29:13,947 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-09 14:29:13,953 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/39d3ee8f-8ff5-446b-9d42-fceff130305a/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
autosynth-176
2020-09-10 01:28:41,214 autosynth [DEBUG] > Running: git checkout autosynth-176
Switched to branch 'autosynth-176'
2020-09-10 01:28:41,220 autosynth [INFO] > Running synthtool
2020-09-10 01:28:41,220 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-10 01:28:41,220 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/176/sponge_log.log
2020-09-10 01:28:41,222 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-10 01:28:41,443 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-176
nothing to commit, working tree clean
2020-09-10 01:28:41,578 synthtool [DEBUG] > Ensuring dependencies.
2020-09-10 01:28:41,589 synthtool [DEBUG] > Cloning googleapis.
2020-09-10 01:28:42,085 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-10 01:28:56,569 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.32.2.zip) = 5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:333:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 14.469s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-10 01:28:56,635 autosynth [ERROR] > Synthesis failed
2020-09-10 01:28:56,635 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a0b20f1 deps: update dependency io.grpc:grpc-bom to v1.31.1 (#16)
2020-09-10 01:28:56,642 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-10 01:28:56,648 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/d9e96217-4f6e-4617-abfe-a9426ee22b75/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
te a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-10 14:37:27,682 autosynth [DEBUG] > Running: git checkout b15c0c042cdea746fc19856527d8baf947c3c220
Note: checking out 'b15c0c042cdea746fc19856527d8baf947c3c220'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b15c0c0 fix: don't attempt to create redundant PR to start tracking obsolete files (#755)
2020-09-10 14:37:27,688 autosynth [DEBUG] > Running: git branch -f autosynth-177
2020-09-10 14:37:27,692 autosynth [DEBUG] > Running: git checkout autosynth-177
Switched to branch 'autosynth-177'
2020-09-10 14:37:27,698 autosynth [INFO] > Running synthtool
2020-09-10 14:37:27,698 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-10 14:37:27,698 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/177/sponge_log.log
2020-09-10 14:37:27,700 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-10 14:37:27,927 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-177
nothing to commit, working tree clean
2020-09-10 14:37:28,068 synthtool [DEBUG] > Ensuring dependencies.
2020-09-10 14:37:28,080 synthtool [DEBUG] > Cloning googleapis.
2020-09-10 14:37:28,424 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-10 14:37:28,620 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
DEBUG: Rule 'com_github_grpc_grpc' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "6a327f757b86469336169580199274ea37fe759489406c4da5d389bee637065d"
DEBUG: Call stack for the definition of repository 'com_github_grpc_grpc' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:113:1
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "5f7c9e6d005efea4f4433020cd11a2b500f92357e1f3285e89ec1aa4bde0a068"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 0.184s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-10 14:37:28,690 autosynth [ERROR] > Synthesis failed
2020-09-10 14:37:28,690 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-10 14:37:28,697 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-10 14:37:28,702 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7bd3109f-291e-465f-9df9-f3d8592f1b05/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git checkout autosynth-177
Switched to branch 'autosynth-177'
2020-09-11 01:34:49,005 autosynth [INFO] > Running synthtool
2020-09-11 01:34:49,005 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-11 01:34:49,005 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/177/sponge_log.log
2020-09-11 01:34:49,007 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-11 01:34:49,228 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-177
nothing to commit, working tree clean
2020-09-11 01:34:49,364 synthtool [DEBUG] > Ensuring dependencies.
2020-09-11 01:34:49,375 synthtool [DEBUG] > Cloning googleapis.
2020-09-11 01:34:49,868 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-11 01:35:04,759 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
Loading: 0 packages loaded
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 14.880s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-11 01:35:04,820 autosynth [ERROR] > Synthesis failed
2020-09-11 01:35:04,820 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-11 01:35:04,826 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 01:35:04,831 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/6d12f39c-c7b3-40ec-b7aa-6dc341b90679/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
20-09-11 14:34:33,788 autosynth [DEBUG] > Running: git checkout autosynth-177
Switched to branch 'autosynth-177'
2020-09-11 14:34:33,793 autosynth [INFO] > Running synthtool
2020-09-11 14:34:33,793 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-11 14:34:33,793 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/177/sponge_log.log
2020-09-11 14:34:33,795 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-11 14:34:33,999 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-177
nothing to commit, working tree clean
2020-09-11 14:34:34,127 synthtool [DEBUG] > Ensuring dependencies.
2020-09-11 14:34:34,138 synthtool [DEBUG] > Cloning googleapis.
2020-09-11 14:34:34,645 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-11 14:34:48,094 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 13.439s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-11 14:34:48,158 autosynth [ERROR] > Synthesis failed
2020-09-11 14:34:48,158 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-11 14:34:48,165 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 14:34:48,170 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ba20ea17-806f-4f7a-aa1b-49f251bc9714/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-12 01:34:59,945 autosynth [INFO] > Running synthtool
2020-09-12 01:34:59,945 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-12 01:34:59,945 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-12 01:34:59,947 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-12 01:35:00,171 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-12 01:35:00,310 synthtool [DEBUG] > Ensuring dependencies.
2020-09-12 01:35:00,322 synthtool [DEBUG] > Cloning googleapis.
2020-09-12 01:35:00,813 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-12 01:35:16,124 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
Loading: 0 packages loaded
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 15.300s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-12 01:35:16,191 autosynth [ERROR] > Synthesis failed
2020-09-12 01:35:16,191 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-12 01:35:16,197 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-12 01:35:16,202 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/264f7de6-6e24-433c-bb50-6531f218875e/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-12 14:36:34,446 autosynth [INFO] > Running synthtool
2020-09-12 14:36:34,446 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-12 14:36:34,446 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-12 14:36:34,448 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-12 14:36:34,683 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-12 14:36:34,823 synthtool [DEBUG] > Ensuring dependencies.
2020-09-12 14:36:34,833 synthtool [DEBUG] > Cloning googleapis.
2020-09-12 14:36:35,341 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-12 14:36:51,135 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 15.784s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-12 14:36:51,191 autosynth [ERROR] > Synthesis failed
2020-09-12 14:36:51,191 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-12 14:36:51,197 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-12 14:36:51,202 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7ce65735-133d-4c7e-8a77-6d503160029a/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
 checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-13 01:35:33,934 autosynth [INFO] > Running synthtool
2020-09-13 01:35:33,934 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-13 01:35:33,935 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-13 01:35:33,937 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-13 01:35:34,162 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-13 01:35:34,306 synthtool [DEBUG] > Ensuring dependencies.
2020-09-13 01:35:34,319 synthtool [DEBUG] > Cloning googleapis.
2020-09-13 01:35:34,840 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-13 01:35:51,058 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 16.205s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-13 01:35:51,105 autosynth [ERROR] > Synthesis failed
2020-09-13 01:35:51,106 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-13 01:35:51,112 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-13 01:35:51,118 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f559274a-d20e-483d-a774-ac38879c04d3/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
20-09-13 14:34:25,021 autosynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-13 14:34:25,026 autosynth [INFO] > Running synthtool
2020-09-13 14:34:25,026 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-13 14:34:25,026 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-13 14:34:25,028 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-13 14:34:25,250 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-13 14:34:25,387 synthtool [DEBUG] > Ensuring dependencies.
2020-09-13 14:34:25,399 synthtool [DEBUG] > Cloning googleapis.
2020-09-13 14:34:25,902 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-13 14:34:39,932 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 14.019s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-13 14:34:39,982 autosynth [ERROR] > Synthesis failed
2020-09-13 14:34:39,982 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-13 14:34:39,988 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-13 14:34:39,993 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5fc7dd4d-91db-45e3-8340-b2dac6ef75ef/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-14 01:34:51,656 autosynth [INFO] > Running synthtool
2020-09-14 01:34:51,656 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-14 01:34:51,656 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-14 01:34:51,658 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-14 01:34:51,875 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-14 01:34:52,009 synthtool [DEBUG] > Ensuring dependencies.
2020-09-14 01:34:52,020 synthtool [DEBUG] > Cloning googleapis.
2020-09-14 01:34:52,510 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-14 01:35:07,480 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
Loading: 0 packages loaded
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 14.959s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-14 01:35:07,540 autosynth [ERROR] > Synthesis failed
2020-09-14 01:35:07,540 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-14 01:35:07,546 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-14 01:35:07,551 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b38c5f85-3845-422c-8a6b-1447439e489d/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
20-09-14 14:34:57,394 autosynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-14 14:34:57,399 autosynth [INFO] > Running synthtool
2020-09-14 14:34:57,399 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-14 14:34:57,399 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-14 14:34:57,401 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-14 14:34:57,614 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-14 14:34:57,746 synthtool [DEBUG] > Ensuring dependencies.
2020-09-14 14:34:57,757 synthtool [DEBUG] > Cloning googleapis.
2020-09-14 14:34:58,249 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-14 14:35:11,764 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 13.505s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-14 14:35:11,811 autosynth [ERROR] > Synthesis failed
2020-09-14 14:35:11,812 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-14 14:35:11,817 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-14 14:35:11,823 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b485cf90-4bd8-4bf6-80f8-696f605f3532/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-15 01:42:15,313 autosynth [INFO] > Running synthtool
2020-09-15 01:42:15,313 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-15 01:42:15,313 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-15 01:42:15,315 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-15 01:42:15,538 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-15 01:42:15,676 synthtool [DEBUG] > Ensuring dependencies.
2020-09-15 01:42:15,689 synthtool [DEBUG] > Cloning googleapis.
2020-09-15 01:42:16,192 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-15 01:42:31,833 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 15.630s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-15 01:42:31,897 autosynth [ERROR] > Synthesis failed
2020-09-15 01:42:31,898 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-15 01:42:31,904 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-15 01:42:31,909 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/19feb6d3-40a4-47b9-8f6c-2e7167da474a/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
20-09-15 14:34:06,828 autosynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-15 14:34:06,833 autosynth [INFO] > Running synthtool
2020-09-15 14:34:06,833 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-15 14:34:06,833 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-15 14:34:06,834 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-15 14:34:07,040 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-15 14:34:07,165 synthtool [DEBUG] > Ensuring dependencies.
2020-09-15 14:34:07,174 synthtool [DEBUG] > Cloning googleapis.
2020-09-15 14:34:07,646 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-15 14:34:20,984 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 13.328s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-15 14:34:21,047 autosynth [ERROR] > Synthesis failed
2020-09-15 14:34:21,047 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-15 14:34:21,053 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-15 14:34:21,057 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5237c30e-85f0-4cf0-89ab-ef47ec1c8572/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
ynth [DEBUG] > Running: git checkout autosynth-178
Switched to branch 'autosynth-178'
2020-09-16 01:39:43,409 autosynth [INFO] > Running synthtool
2020-09-16 01:39:43,409 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-16 01:39:43,409 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/java-common-protos/178/sponge_log.log
2020-09-16 01:39:43,412 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-16 01:39:43,655 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/java-common-protos/synth.py.
On branch autosynth-178
nothing to commit, working tree clean
2020-09-16 01:39:43,810 synthtool [DEBUG] > Ensuring dependencies.
2020-09-16 01:39:43,822 synthtool [DEBUG] > Cloning googleapis.
2020-09-16 01:39:44,340 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-16 01:40:01,900 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
Loading: 0 packages loaded
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 17.547s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-16 01:40:01,998 autosynth [ERROR] > Synthesis failed
2020-09-16 01:40:01,999 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at a51daab deps: update dependency com.google.protobuf:protobuf-bom to v3.13.0 (#26)
2020-09-16 01:40:02,006 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-16 01:40:02,012 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4448a658-3945-4c4a-ad8c-c27fbe9f8fdd/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
o commit, working tree clean
2020-09-16 14:37:26,228 synthtool [DEBUG] > Ensuring dependencies.
2020-09-16 14:37:26,245 synthtool [DEBUG] > Cloning googleapis.
2020-09-16 14:37:26,821 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-16 14:37:44,108 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-csharp/archive/v1.3.1.zip) = 1118448d4c3db2a443e0bc3bf240de749adbae3311703e264d14213041b6f1c3
DEBUG: Rule 'gapic_generator_csharp' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "1118448d4c3db2a443e0bc3bf240de749adbae3311703e264d14213041b6f1c3"
DEBUG: Call stack for the definition of repository 'gapic_generator_csharp' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:327:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 17.270s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-16 14:37:44,200 autosynth [ERROR] > Synthesis failed
2020-09-16 14:37:44,200 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at eb88e58 build(deps): update dependency org.apache.maven.plugins:maven-project-info-reports-plugin to v3.1.1 (#30)
2020-09-16 14:37:44,207 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-16 14:37:44,213 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/2a25115a-a3a1-47ff-b9e8-1633953258a0/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
nch autosynth-180
nothing to commit, working tree clean
2020-09-17 01:35:20,225 synthtool [DEBUG] > Ensuring dependencies.
2020-09-17 01:35:20,237 synthtool [DEBUG] > Cloning googleapis.
2020-09-17 01:35:20,768 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-17 01:35:35,368 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-csharp/archive/v1.3.1.zip) = 1118448d4c3db2a443e0bc3bf240de749adbae3311703e264d14213041b6f1c3
DEBUG: Rule 'gapic_generator_csharp' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "1118448d4c3db2a443e0bc3bf240de749adbae3311703e264d14213041b6f1c3"
DEBUG: Call stack for the definition of repository 'gapic_generator_csharp' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:327:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 14.588s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-17 01:35:35,447 autosynth [ERROR] > Synthesis failed
2020-09-17 01:35:35,447 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at eb88e58 build(deps): update dependency org.apache.maven.plugins:maven-project-info-reports-plugin to v3.1.1 (#30)
2020-09-17 01:35:35,454 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-17 01:35:35,460 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/24f3fe08-442c-4c71-bcc4-22bdd4d1d3a3/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

Autosynth is still having trouble generating java-common-protos. :sob:

Here's the output from running `synth.py`:

```
nch autosynth-181
nothing to commit, working tree clean
2020-09-17 14:51:27,122 synthtool [DEBUG] > Ensuring dependencies.
2020-09-17 14:51:27,134 synthtool [DEBUG] > Cloning googleapis.
2020-09-17 14:51:27,621 synthtool [DEBUG] > Generating code for: //google/api:google-api-java.
2020-09-17 14:51:41,228 synthtool [ERROR] > Failed executing bazel --max_idle_secs=240 build //google/api:google-api-java:

Loading: 
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gax-java/archive/v1.58.2.zip) = 4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161
DEBUG: Rule 'com_google_api_gax_java' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4638977c21dcb471914d051da5fcd767cc28f2b84086acffd61adddf60136161"
DEBUG: Call stack for the definition of repository 'com_google_api_gax_java' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:146:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator/archive/v2.4.6.zip) = e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b
Loading: 0 packages loaded
DEBUG: Rule 'com_google_api_codegen' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e7180d441d12864666f7d324986e2e4c7e6d38e3621dc91161f1e2299da4220b"
DEBUG: Call stack for the definition of repository 'com_google_api_codegen' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:77:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-python/archive/v0.33.0.zip) = 678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed
DEBUG: Rule 'gapic_generator_python' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "678005370af4f7ecd6fb6601aa32712c3998fa1ef151377686dd3f78705fe7ed"
DEBUG: Call stack for the definition of repository 'gapic_generator_python' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:225:1
Loading: 0 packages loaded
Loading: 0 packages loaded
INFO: SHA256 (https://github.com/googleapis/gapic-generator-go/archive/v0.15.1.tar.gz) = e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0
DEBUG: Rule 'com_googleapis_gapic_generator_go' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e69ee7264f5092cbfcf0a94c7732a55be4ba4fa5f29bb727c75fd1fa88cc39d0"
DEBUG: Call stack for the definition of repository 'com_googleapis_gapic_generator_go' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:262:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-typescript/archive/v1.1.0.tar.gz) = 37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df
DEBUG: Rule 'gapic_generator_typescript' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "37799a8ce876061ffb27d79b69d830f11bf2574616a2d7f9cc1e1530e48686df"
DEBUG: Call stack for the definition of repository 'gapic_generator_typescript' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:281:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-csharp/archive/v1.3.1.zip) = 1118448d4c3db2a443e0bc3bf240de749adbae3311703e264d14213041b6f1c3
DEBUG: Rule 'gapic_generator_csharp' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "1118448d4c3db2a443e0bc3bf240de749adbae3311703e264d14213041b6f1c3"
DEBUG: Call stack for the definition of repository 'gapic_generator_csharp' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:327:1
INFO: SHA256 (https://github.com/googleapis/gapic-generator-ruby/archive/9cc95d8f4e05bdfe4ea9c67e6ca670f27c01c8f2.zip) = cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41
DEBUG: Rule 'gapic_generator_ruby' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "cfa78149b54f69f991b9f5fd1f5f8c68ead8b8b52679a9bd48afdfb3f5a32e41"
DEBUG: Call stack for the definition of repository 'gapic_generator_ruby' which is a http_archive (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/bazel_tools/tools/build_defs/repo/http.bzl:296:16):
 - <builtin>
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:341:1
ERROR: Skipping '//google/api:google-api-java': no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//google/api:google-api-java': target 'google-api-java' not declared in package 'google/api' defined by /home/kbuilder/.cache/synthtool/googleapis/google/api/BUILD.bazel
INFO: Elapsed time: 13.597s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/java-common-protos/synth.py", line 34, in <module>
    bazel_target=f"//google/{path}:{target}",
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 62, in java_library
    service, version, "java", tar_strip_components=0, **kwargs
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/api:google-api-java']' returned non-zero exit status 1.
2020-09-17 14:51:41,312 autosynth [ERROR] > Synthesis failed
2020-09-17 14:51:41,312 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at eb88e58 build(deps): update dependency org.apache.maven.plugins:maven-project-info-reports-plugin to v3.1.1 (#30)
2020-09-17 14:51:41,319 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-17 14:51:41,325 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/a89ada08-4dfb-47d5-b8f7-042aafb7f10f/targets/github%2Fsynthtool;config=default/tests;query=java-common-protos;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -14,32 +14,81 @@
 
 """This script is used to synthesize generated parts of this library."""
 
-import synthtool as s
-from synthtool import gcp
-from synthtool.sources import git
-import synthtool.languages.java as java
+import os
 from pathlib import Path
+import tempfile
+from typing import Union
 
-gapic = gcp.GAPICBazel()
-googleapis = Path("/home/chingor/code/googleapis")
+import synthtool as s
+from synthtool.languages import java
+from synthtool.sources import git
+from synthtool import logger, shell
 
-proto_targets = ["api", "cloud/audit", "geo/type", "logging/type", "longrunning", "rpc", "rpc/context", "type"]
-for path in proto_targets:
-    target = path.replace("/", "-")
-    target = f"google-{target}-java"
-    library = gapic.java_library(
-        service=target,
-        version="unused",
-        proto_path=f"google/{path}:", #google-iam-{version}-logging-java',
-        bazel_target=f"//google/{path}:{target}",
+GOOGLEAPIS_URL = git.make_repo_clone_url("googleapis/googleapis")
+
+logger.debug("Cloning googleapis.")
+googleapis = git.clone(GOOGLEAPIS_URL)
+
+def bazel_build(target: str, cwd: Union[Path, str]) -> Path:
+    """Build a bazel target and return the output build directory."""
+    old_cwd = os.getcwd()
+    os.chdir(str(cwd))
+
+    bazel_run_args = [
+        "bazel",
+        "--max_idle_secs=240",
+        "build",
+        target,
+    ]
+
+    logger.debug(f"Generating code for: {target}.")
+    shell.run(bazel_run_args)
+
+    output_dir = Path(f"bazel-bin{os.path.sep}{target[2:].split(':')[0]}").resolve()
+    os.chdir(old_cwd)
+
+    return output_dir
+
+def build_proto(target):
+    """Build a proto build target and copy all generate source files."""
+    output = bazel_build(
+        target=target,
+        cwd=googleapis,
     )
 
-    library = library / target
-    java.fix_proto_headers(library / f"proto-{target}")
-    s.copy(library / f"proto-{target}" / "src", "proto-google-common-protos/src", required=True)
+    src_output = Path(tempfile.mkdtemp())
+    for proto_jar in output.glob("*-speed-src.jar"):
+        logger.debug(f"unzipping: {os.path.basename(proto_jar)}")
+        shell.run(["unzip", "-o", proto_jar, "-d", src_output / "src"])
 
-    java.fix_grpc_headers(library / f"grpc-{target}", package_name=None)
-    s.copy(library / f"grpc-{target}" / "src", "grpc-google-common-protos/src")
+    java.fix_proto_headers(src_output)
+    s.copy(src_output / "src/com", "proto-google-common-protos/src/main/java/com")
+
+def build_grpc(target):
+    """Build a grpc build target and copy all generate source files."""
+    output = bazel_build(
+        target=target,
+        cwd=googleapis,
+    )
+
+    src_output = Path(tempfile.mkdtemp())
+    for proto_jar in output.glob("*grpc-src.jar"):
+        logger.debug(f"unzipping: {os.path.basename(proto_jar)}")
+        shell.run(["unzip", "-o", proto_jar, "-d", src_output / "src"])
+
+    java.fix_grpc_headers(src_output, "")
+    s.copy(src_output / "src/com", "grpc-google-common-protos/src/main/java/com")
+
+build_proto("//google/api:api_java_proto")
+build_proto("//google/cloud/audit:audit_java_proto")
+build_proto("//google/geo/type:viewport_java_proto")
+build_proto("//google/logging/type:type_java_proto")
+build_proto("//google/longrunning:longrunning_java_proto")
+build_proto("//google/rpc:rpc_java_proto")
+build_proto("//google/rpc/context:attribute_context_java_proto")
+build_proto("//google/type:type_java_proto")
+
+build_grpc("//google/longrunning:longrunning_java_grpc")
 
 java.format_code("proto-google-common-protos/src")
 java.format_code("grpc-google-common-protos/src")

